tests/rt.sh

```diff
@@ -355,11 +355,6 @@ SINGLE_NAME=''
 
 TESTS_FILE='rt.conf'
 
-if [[ $MACHINE_ID = orion.* ]]; then
```

Comment:
Should we do the same with Cheyenne and Stampede in the future?

Reply:
I think so, but I think we may need an option to define which tests we will not run on a certain platforms.

Reply:
Yes, I remember we talked about this. Something like a `-cheyenne.intel`, i.e. the `-` negates running the test on that platform. Same for `COMPILE` and `RUN` lines. Not that for compile steps that work on all platforms, you can just use one `COMPILE` line and leave the machine column empty.